atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/TimestampDecoratingTimelockService.java

```diff
@@ -44,6 +44,11 @@ public TimestampRange getFreshTimestamps(int numTimestampsRequested) {
         return decoratedTimestamps.getFreshTimestamps(numTimestampsRequested);
     }
 
+    @Override
+    public void close() {
+        delegate.close();
```

Comment:
I think you need to close `decoratedTimestamps` if it is closeable - in practice this is going to be a `RequestBatchingTimestampService` which needs to be closed to shut down the disruptor stuff. 

It's a bit sad. Maybe this class can take a `CloseableTimestampService` instead?

Reply:
even better, I removed `TimestampDecoratingTimelockService`, apparently we don't use it anymore (I guess we were using it to make request batching runtime configurable?)